src/ng/directive/ngOptions.js

```diff
@@ -624,7 +629,7 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
         // the option list in listbox style, i.e. the select is [multiple], or specifies a [size].
         // See https://github.com/angular/angular.js/issues/11314 for more info.
         // This is unfortunately untestable with unit / e2e tests
-        if (option.label !== element.label) {
+        if (option.label !== undefined) {
```

Comment:
element.label always will be undefined (because we create a new empty option element each time), then it's useless to read element.label value if we know that its value is undefined.